components/Premium/PremiumFeatures/PremiumFeatures.tsx

```diff
@@ -22,62 +29,143 @@ function PremiumFeatures() {
-                        <td className={styles.featureColumn}>
-                            List of low supply items (<Link href="/lowSupply">here</Link>)
+                        <td className={styles.featureColumn}>Discord role</td>
+                        <td className={styles.premiumProductColumn}>-</td>
+                        <td className={styles.premiumProductColumn}>-</td>
+                        <td className={styles.premiumProductColumn}>Flipper</td>
+                        <td className={styles.premiumProductColumn}>Sniper</td>
+                    </tr>
+                    <tr>
+                        <td className={styles.featureColumn}>Max. Notifications</td>
+                        <td className={styles.premiumProductColumn}>3</td>
+                        <td className={styles.premiumProductColumn}>10</td>
+                        <td className={styles.premiumProductColumn}>100</td>
+                        <td className={styles.premiumProductColumn}>{numberWithThousandsSeperators(1000)}</td>
+                    </tr>
+                    <tr>
+                        <td className={styles.featureColumn}>Chat colors</td>
+                        <td className={styles.premiumProductColumn} style={{ color: 'gray', fontWeight: 'bold' }}>
+                            Gray
+                        </td>
+                        <td className={styles.premiumProductColumn} style={{ color: 'white', fontWeight: 'bold' }}>
+                            White
+                        </td>
+                        <td className={styles.premiumProductColumn} style={{ color: '#32de84', fontWeight: 'bold' }}>
+                            Green
+                        </td>
+                        <td className={styles.premiumProductColumn} style={{ color: '#ffaa00', fontWeight: 'bold' }}>
+                            Gold
                         </td>
-                        <td>{xIcon}</td>
-                        <td>{checkIcon}</td>
                     </tr>
                     <tr>
-                        <td className={styles.featureColumn}>Discord role "Flipper" (on request)</td>
-                        <td>{xIcon}</td>
-                        <td>{checkIcon}</td>
+                        <td className={styles.featureColumn}>
+                            Average flip receive time
+                            <Tooltip
+                                content={
+                                    <span style={{ marginLeft: '5px' }}>
+                                        <HelpIcon />
+                                    </span>
+                                }
+                                type="hover"
+                                tooltipContent={
+                                    <p>
+                                        Hypixel updates once every 60 seconds. After the we receive the new auctions this is how long it will take for you to
+                                        get the flips.
```

Comment:
```suggestion
                                        The Hypixel Auctions API updates once every 60 seconds. 
                                        After we were able to load new auctions, this is how long it will take until they are shown to you. (Parsing auctions, finding references, comparing to determine profit, distributing and filtering and seinding to you) 
```